src/configs/index.js

```diff
@@ -48,9 +48,54 @@ export const TIME_OPTIONS = {
   day: 'numeric'
 };
 
+
 export const LANGUAGES_LIST = [
   { lang: 'UA', value: 0 },
   { lang: 'EN', value: 1 }
+
+export const CONFIRM_EMAIL = [
```

Comment:
Please, create a file with translations inside the `translations` folder, this code should not be here, in config file

Reply:
moved

Reply:
created